conf/service/event_analyzer.service

```diff
@@ -13,18 +13,15 @@
 # about this software or licensing, please email opensource@seagate.com or
 # cortx-questions@seagate.com.
 
+# TODO: convert event_analyser.service to event_analyser@consumer_id.service for scaling
 [Unit]
 Description=HA event analyzer daemon process
-After=syslog.target
 
 [Service]
 Type=simple
-ExecStartPre=/usr/bin/sleep 5
 ExecStart=/usr/bin/event_analyzerd
-TimeoutStopSec=7sec
```

Comment:
You should reconsider this.

Reply:
keeping 30 sec considering consul call can take some time